packages/graphback-datasync/src/util.ts

```diff
@@ -39,3 +44,59 @@ export const DataSyncFieldNames = {
   lastUpdatedAt: '_lastUpdatedAt',
   deleted: '_deleted'
 }
+
+
+
+export interface DataSyncModelConflictConfig {
+  enabled: boolean
+  conflictResolution?: ConflictResolutionStrategy
+}
+
+export interface DataSyncModelConfigMap {
+  [modelName: string]: DataSyncModelConflictConfig
+}
+
+export interface ConflictResolutionStrategy {
+  resolveUpdate(conflict: ConflictMetadata): any
+  resolveDelete(conflict: ConflictMetadata): any
+}
+
+export const ServerSideWins: ConflictResolutionStrategy = {
+  resolveUpdate(conflict: ConflictMetadata): any {
+    const { serverData, serverDiff, clientDiff } = conflict;
+
+    if (serverData[DataSyncFieldNames.deleted] === true) {
+      throw new ConflictError(conflict);
+    }
+
+    const resolved = Object.assign(serverData, clientDiff, serverDiff);
+
+    return resolved
+  },
+  resolveDelete(conflict: ConflictMetadata): any {
+
+    throw new ConflictError(conflict);
```

Comment:
We might need to know what operation thrown conflict (in case someone will start batching requests on client)

Separate issue probably?